docs/canvas/canvas-function-reference.asciidoc

```diff
@@ -374,6 +374,17 @@ Clears the _context_, and returns `null`.
 *Returns:* `null`
 
 
+[float]
+[[clog_fn]]
+=== `clog`
+
+This function is for debug purpose. It outputs the _context_ in the console.
```

Comment:
Could you switch the definition and the purpose? For example:

Outputs the _context_ in the Console. Use this function for debugging.